doc/source/whatsnew/v1.1.0.rst

```diff
@@ -642,7 +642,7 @@ ExtensionArray
 ^^^^^^^^^^^^^^
 
 - Fixed bug where :meth:`Serires.value_counts` would raise on empty input of ``Int64`` dtype (:issue:`33317`)
--
+- Fixed bug in :meth:`GenericArrayFormatter._format_strings` that caused :meth:`Series.__repr__()` to crash for some extension types (:issue:`33770`).
```

Comment:
can you make this more user-readable, e.g. this is an internal function.

Reply:
Commit 75d22c8 rewords that description to describe the problem in terms of public APIs.